rust/src/lib/ifaces/base.rs

```diff
@@ -218,3 +222,11 @@ fn default_state() -> InterfaceState {
 fn default_iface_type() -> InterfaceType {
     InterfaceType::Unknown
 }
+
+fn is_option_string_empty(data: &Option<String>) -> bool {
+    if let Some(s) = data {
+        s.is_empty()
+    } else {
+        false
+    }
```

Comment:
Can data be `None`? Since it is an option I assume yes, so this function will serialize it if so.. is that correct at all? I think the description can be None..

Reply:
When user ask for `Some("".to_string())`, it means user would like to delete the description. For the seek of verification, we don't show empty description.